dask/array/tests/test_linalg.py

```diff
@@ -18,8 +18,6 @@ def test_tsqr_regular_blocks():
     data = da.from_array(mat, chunks=(10, n), name='A')
 
     q, r = tsqr(data)
-    q = np.array(q)
-    r = np.array(r)
```

Comment:
Thank you for cleaning these up, even though they're unrelated to your work here.  I've gone through the test suite and replaced the explicit conversion to numpy arrays with dask array functions.  I hope you don't mind that I pushed to your branch.